library/src/main/java/com/xfinity/blueprint/model/Component.kt

```diff
@@ -16,6 +16,14 @@ import com.xfinity.blueprint.presenter.DefaultComponentPresenter
 
 interface ComponentModel
 
+/**
+ * StaticComponent are Components that have no Model or Presentation, just a layout that needs to be inflated and added.
+ * Having a Model with no data will cause the DiffUtil Callback to always flag a Component as "new" and re-add it whenever
+ * the screen is presented, which can cause blinking and other UI artifacts.  We'll use StaticComponent to add a default
+ * Model that contains the viewType so that these types of Components are ignored in the Diff calculation.
+ */
+data class StaticComponentModel(val viewType: Int) : ComponentModel
```

Comment:
One note - you should test this with a setup that has multiple static components of the same type. You could potentially see weird "slide" behaviors there, especially if one of the static components is being added or removed, as it'll end up flagging as "contents same".